DEV.md

```diff
@@ -31,7 +31,9 @@ it's corresponding code path from the above table.
 2. Install required packages:
 
 ```sh
-$ apt install git xen-system-amd64 build-essential libclang-dev musl-tools flex bison libelf-dev libssl-dev bc protobuf-compiler libprotobuf-dev squashfs-tools erofs-utils
+$ sudo apt install git xen-system-amd64 build-essential \
```

Comment:
I wouldn't recommend including sudo in the commands here, easy enough to add by hand and requires consciously invoking. Everything else looks good